resources/views/errors/503.blade.php

```diff
@@ -2,7 +2,7 @@
 <html>
     <head>
         <title>Be right back.</title>
-
+        <link rel="icon" href="../../favicon.ico">
```

Comment:
This appears to be the wrong folder structure.

Look here for Laravel's folder structure: https://laravel.com/docs/master/structure

Files in the public directory will appear in the root of the domain, i.e.

public/favicon.ico -> http://members.cumsa.org/favicon.ico

As it stands your favicon.ico is not publicly accessible, and ../../favicon.ico -> /favicon.ico.